### PR TITLE
Add label and issuer to otp config

### LINKED ISSUE #4215: OTP Label is username instead of wallabag or instance
### Issue details

QR code URI is `otpauth://totp/%username%?secret=%secret%`. Instead of username [it should be a label](https://github.com/google/google-authenticator/wiki/Key-Uri-Format) to know about the service or instance, something like `wallabag` maybe including the instance, such as `wallabag (wallabag.it)`.

### Environment

`wallabag.it`

### Steps to reproduce/test case

Setup OTP.

### PR DESCRIPTION
**Pull request by tcitworld**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | 
| Documentation | no
| Translation   | no
| CHANGELOG.md  | no
| License       | MIT

Closes #4215

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -208,6 +208,8 @@ scheb_two_factor:
 
     google:
         enabled: "%twofactor_auth%"
+        label: wallabag
```
Comment (Kdecherf) [inline app/config/config.yml]: Travis does not seem happy with this change:

  https://travis-ci.org/github/wallabag/wallabag/jobs/667283180#L738

Comment (tcitworld) [reply app/config/config.yml]: It's called `server_name` in version 4.4 that we use.

Comment (tcitworld) [inline app/config/config.yml]:
```suggestion
        server_name: wallabag
```

**Comment by tcitworld** [comment]
Needs more debug.

**Comment by Kdecherf** [comment]
This PR creates a new parameter named `server_name`.
It is used as a user-friendly name, like a company name (_e.g. 'Your wallabag instance' by default_), for the issuer of the TOTP qrcode.

A parameter like `domain_name` can't be used as the issuer here because it contains forbidden chars.

**Comment by Kdecherf** [comment]
I will need to squash this PR before merging it, it is ready for review though.

**Review by Kdecherf — DISMISSED** [review]

**Review by nicosomb — DISMISSED** [review]

**Review by j0k3r — COMMENTED** [review]
Looks great, you can squash :+1: 
Also don't forget to update the doc about that new parameter.

**Comment by Kdecherf** [comment]
@j0k3r rebase and squash done @j0k3r, I'll open a PR on the doc and docker repositories to add some info on this new parameter.

**Review by j0k3r — APPROVED** [review]

**Pull request merged**